### PR TITLE
Adding QPE Benchmark

### PR DESCRIPTION
**Pull request by Yelissal**

### CODE REVIEW

**Review by Lementknight — APPROVED** [review]
Looks good to me. Way to go, Yelissa!

**Review by Lementknight — CHANGES_REQUESTED** [review]
Remove the extra qasm folder you have in the `red_queen` directory.

File: red_queen/games/applications/run_qpe.py
Comment (danielleodigie) [inline red_queen/games/applications/run_qpe.py:2]:
```suggestion
# Part of Red Queen Project.  This file is distributed under the Apache 2.0 License.
```

**Review by danielleodigie — COMMENTED** [review]
Looks good! One tiny thing with the licenses

**Review by keilydluna — APPROVED** [review]
Fidelities look good! Don't forget its Apache 2.0 License.

**Review by danielleodigie — APPROVED** [review]
LGTM!

**Review by Lementknight — APPROVED** [review]
Looks good to me. Great job, Yelissa!

**Review by mtreinish — APPROVED** [review]
This LGTM, thanks for sticking with it and making all the updates. I'm ready to merge but I'd like to wait until we get #29 merged so we can get a test run for this in CI before we merge.

**Pull request merged**
